api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db.hibernate;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.ConceptMapType;
+import org.openmrs.Order;
+import org.openmrs.OrderGroup;
+import org.openmrs.api.builder.OrderBuilder;
+import org.openmrs.api.db.ConceptDAO;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.openmrs.test.Verifies;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Date;
+import java.util.LinkedHashSet;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests the saving of orders as part of the OrderGroup
+ */
+public class HibernateOrderDAOTest extends BaseContextSensitiveTest {
+	
+	@Autowired
+	private HibernateOrderDAO dao;
+	
+	private static final String ORDER_SET = "org/openmrs/api/include/OrderSetServiceTest-general.xml";
+	
+	@Before
+	public void setUp() throws Exception {
+		executeDataSet(ORDER_SET);
+	}
+
+	/**
+	 * @see {@link HibernateOrderDAO#saveOrderGroup(OrderGroup)}
+	 * @throws Exception
+	 */
+	@Test
+	@Verifies(value = "saves the order group ", method = "saveOrderGroup(OrderGroup)")
+	public void saveOrderGroup_shouldSaveOrderGroup() throws Exception {
```

Comment:
This test has been removed from orderServiceTest.
Here were are just checking that saving of the orderGroup happens separately and it doesn't save orders along with it.